api/src/main/proto/stellarstation/api/v1/monitoring/monitoring.proto

```diff
@@ -34,6 +34,8 @@ option java_package = "com.stellarstation.api.v1.monitoring";
 //
 // A transmitter takes a bitstream and applies signal processing to create the final waveform sent
 // to a satellite.
+// TODO: Add parameters which describes whether modulation, idle pattern and IF sweep is enabled.
```

Comment:
Don't think there should be TODOs in the public API, but let's wait for a second opinion.

Reply:
That's make sense. I should manage them under Github Issue. I will create them..